install-dev/data/db_structure.sql

```diff
@@ -1349,6 +1349,7 @@ CREATE TABLE `PREFIX_order_detail` (
   `product_ean13` varchar(13) DEFAULT NULL,
   `product_isbn` varchar(32) DEFAULT NULL,
   `product_upc` varchar(12) DEFAULT NULL,
+  `product_mpn` varchar(32) DEFAULT NULL,
```

Comment:
Shouldn't the varchar's length be 40 here? Since in php the max length is set to 40?

Reply:
Yes!